Class-11-(Model-Classes-and-Model-Binding).md

```diff
@@ -0,0 +1,33 @@
+# Class 10: Models Classes and Model Binding
+
+## In the prep work this week, the students learned
+
+1. The role of *models* in the MVC pattern
+1. The distinction between the functionality of models versus controllers and data services
+1. The data handling practice of assigning a unique id to objects created
+1. The use and benefits of model binding
+1. The job of the ``@ModelAttribute`` annotations
+
+## Announcements
+
+1. Check with your course manager for any important announcements.
+1. Assignment 3 due date is approaching. With the knowledge they now have about model classes, they will be able to complete the assignment.
+
+## Large Group Thyme (Instructor)
+
+### Class 11 Topics That Require Careful Attention
+
+1. With all of the pieces of MVC in place, a review of all of the components and their relationships to each other and the client and the server is a good starting-off point.
+1. It may be tempting for students to associate models with data storage. In fact, we do use a model for temporary data storage in the example repo, ``coding-events``. Emphasize that this aspect will be refactored and models are not raw data in a proper mvc app.
```

Comment:
I would refer to the data service class w/o calling it a model. This is probably more correct, and will make the distinction between models and services clearer.